docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddUsings)]
+
+Create global fields to hold the paths to the various assets, and global variables for the `LabelTokey`,`ImageReal`, and  `PredictedLabelValue`:
+
+* `_assetsPath` has the path to the assets.
+* `_trainTagsTsv` has the path to the training image data tags tsv file.
+* `_predictTagsTsv` has the path to the prediction image data tags tsv file.
+* `_trainImagesFolder` has the path to the images used to train the model.
+* `_predictImagesFolder` has the path to the images to be classified by the trained model.
+* `_inceptionPb` has the path to the pre-trained model to be reused to train the model.
+* `_inputImageClassifierZip` has the path where the trained model is loaded from.
+* `_outputImageClassifierZip` has the path where the trained model is saved.
+* `LabelTokey` is the `Label` value mapped to a key.
+* `ImageReal`  is the column containing the predicted label value. 
+* `PredictedLabelValue` is the column containing the predicted label value.
+
+Add the following code to the line right above the `Main` method to specify those paths and the other variables:
+
+[!code-csharp[DeclareGlobalVariables](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#DeclareGlobalVariables)]
+
+Create some classes for your input data, inspections, and predictions. Add a new class to your project:
+
+1. In **Solution Explorer**, right-click the project, and then select **Add** > **New Item**.
+
+1. In the **Add New Item** dialog box, select **Class** and change the **Name** field to *ImageNetData.cs*. Then, select the **Add** button.
+
+    The *ImageNetData.cs* file opens in the code editor. Add the following `using` statement to the top of *ImageNetData.cs*:
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/tutorials/TransferLearningTF/ImageNetData.cs#AddUsings)]
+
+Remove the existing class definition and add the following code, which has two classes `ImageNetData` and `ImageNetPipeline`, to the *ImageNetData.cs* file:
+
+[!code-csharp[DeclareGlobalVariables](../../../samples/machine-learning/tutorials/TransferLearningTF/ImageNetData.cs#DeclareTypes)]
+
+`ImageNetData` is the input image data class and has the following <xref:System.String> fields:
+
+* `ImagePath` contains the path for the image file
+* `Label` contains a value for the image label
+
+`ImageNetData` also has a method (`ReadFromCsv`) to read the image data `.csv` file and load it into an `ImageNetData` object.
+
+`ImageNetPipeline` is the class used for inspecting column values after the model has been trained. It has a `string` (`ImagePath`) for the image path. The `Label` is used to create and train the model, and it's also used to evaluate the model. The `PredictedLabelValue` is used during prediction and evaluation. For evaluation, an input with training data, the predicted values, and the model are used.
```

Comment:
Hard to follow. 
What is a column value? 
How is the label being used to create and train a model and also for evaluation in the same class?